README.md

```diff
@@ -1,13 +1,15 @@
 # Release Information
 
-- **Version**:  2.1.0
+- **Version**:  3.0.0
 - **Certified**: Yes
 - **Publisher**: Fortinet
 - **Compatible Version**: FortiSOAR v7.6.1 and later
 - [Release Notes](./release_notes.md)
 
 > [!Important]
-> After an upgrade, you must re-run the **Threat Intel Management** configuration wizard. Before running the wizard, ensure that the NIST NVD connector is configured.
+> After an upgrade:
```

Comment:
Content Hub link should be point to https://fortisoar.contenthub.fortinet.com//detail.html?entity=threat-intel-management&version=3.0.0&type=solutionpack

Reply:
Why? User is already in the threat intel management doc.

Reply:
Updated the link to https://fortisoar.contenthub.fortinet.com//list.html?contentType=solutionpack&searchContent=threat%20intel%20management